python/src/opendp/context.py

```diff
@@ -383,7 +383,7 @@ def compositor(
         split_evenly_over: Optional[int] = None,
         split_by_weights: Optional[list[float]] = None,
         domain: Optional[Domain] = None,
-        margins: Optional[MutableMapping[tuple[str, ...], Margin]] = None,
+        margins: Optional[Union[MutableMapping[tuple[str, ...], Margin], Sequence[tuple[tuple[Any, ...], Margin]]]] = None,
```

Comment:
I worry about introducing unions, but if this is the right direction, the `:param:` doc below needs to be updated to explain the new option.